README.md

```diff
@@ -412,7 +412,7 @@ Caution: Devise Controllers inherit from ApplicationController. If your app uses
 
 ### Test helpers
 
-Devise includes some test helpers for functional specs. In order to use them, you need to include Devise in your functional tests by adding the following to the bottom of your `test/test_helper.rb` file:
+Devise includes some test helpers for functional specs. In order to use them, you need to include Devise in your functional tests by adding the following to the bottom of your `test/test_helper.rb` file (make sure you place it out of scope of ActiveSupport::TestCase which is the default class inside of 'test/test_helper.rb):
```

Comment:
Yep, I have just done that, let me know if there is anything else I can do. Thanks for taking the time to have a look at my PR. I am a fairly new developer and am looking to contribute to more to open source, so hopefully you will see some more PR from me on Devise.